### DISCUSSION TITLE
First release is ready, version 0.2-beta

### DISCUSSION THREAD

**Discussion by kaixxx** [kaixxx/noScribe#1] [Announcements]
You can download the windows installer from here:
https://drive.switch.ch/index.php/s/EIVup04qkSHb54j > go to folder "0.2b"

**Comment by BabyFnord** [comment]
_Very_ promising project, thanks for making it accessible to the public! Having said that, I'd have to wait for a version compatible with macos (ye olde 10.14.x, in my case). Maybe you like to consider a libre packet manager like [Homebrew](https://brew.sh/) for distribution, which could take care for the required dependencies. 

As I see you're affiliated with the IfS in Frankfurt, I could offer to meet up at the campus for testing on a Mac that has Homebrew and the usual Python, Ruby, gcc etc. toolchain installed (though pls note, I'm just a PhD candidate with tons of interview audio to be transcribed, not a dev …).

**Reply by kaixxx** [reply]
Thank you! 
If you have python etc. already installed, you could try running noScribe from source. It's not very difficult. See my hints here: https://github.com/kaixxx/noScribe#development-and-contribution The biggest problem will probably be to find a compiled version of whisper.cpp or to compile one yourself.

There are however a few other native macos apps already available which are based on the same AI. They are not so specialized on transcribing interviews (no speaker detection), but might still work for you. Have a look here: 
https://github.com/ggerganov/whisper.cpp/discussions/categories/show-and-tell

Thank you for offering me your machine as a test bed! I might come back to this later.

**Comment by mikenoname-github** [comment]
I would love to try your product, but every attempt at downloading it fails with "unknown network errors".

My Internet connection is fine and all other websites seem just fine, so the issue seems to be on the source server.

Do you have a mirror download site?

**Reply by kaixxx** [reply]
Hmm, seems fine for me. It's a very large download. You may try freedownloadmanager or something similar
